### PR TITLE
Fixing the map display in IE 11

### PR DESCRIPTION
**Pull request by flacoman91**
IE11 support of flex box is not good, so it must be fixed.

No breaking changes with Chrome:
Chrome

IE 11 fixes, using the breakpoints in the original responsive [ticket](https://github.com/cfpb/ccdb5-ui/pull/188)
widths: 1200, 901, 899, 749, 599, 375
IE11

### CODE REVIEW

**Comment by JeffreyMFarley** [comment]
Unfortunately, here is the matrix for checking the views

Browser|1200 px|1000 px|751 px|749 px|599 px|< 400 px
---|---|---|---|---|---|---
Chrome|||Filters|No filters||No Values
IE 11|

**Comment by flacoman91** [comment]
> Unfortunately, here is the matrix for checking the views
> 
> Browser	1200 px	1000 px	751 px	749 px	599 px	< 400 px
> Chrome			Filters	No filters		No Values
> IE 11

back to the drawing board for me then.

**Comment by flacoman91** [comment]
Because pics or it didn't happen:

IE11-751
IE11-1200
IE11-1000

IE11-399
IE11-599
IE11-749

**Comment by flacoman91** [comment]
Chrome 
Chrome

File: src/TileChartMap.less [hunk 1]
```diff
@@ -146,10 +146,6 @@ x/y positioning on the SVG elements.
         .legend-box {
             stroke: @gray;
         }
-
-        .legend-text {
-            transform: translate(3px, -3px);
-        }
```
Comment (JeffreyMFarley) [inline src/TileChartMap.less]: In the screenshots, the tile text looks shifted.  Do we need to restore this?

File: src/TileChartMap.less [hunk 2]
```diff
@@ -249,6 +245,11 @@ x/y positioning on the SVG elements.
         display: flex;
         flex-direction: row;
         flex-wrap: wrap;
+        // IE 11 fixes
+        @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+            display: inline-block;
+            float: left;
+        }
```
Comment (JeffreyMFarley) [inline src/TileChartMap.less]: :+1:

File: src/TileMap/index.js [hunk 3]
```diff
@@ -169,9 +169,17 @@ export function getColorByValue( value, bins ) {
  * @returns {string} html output
  */
 export function tileFormatter() {
+  let iePatch = ''
+  if ( navigator.userAgent.indexOf( 'MSIE' ) !== -1 || navigator.appVersion.indexOf( 'Trident/' ) > -1 ) {
+    iePatch = '<br />'
+  }
```
Comment (JeffreyMFarley) [inline src/TileMap/index.js]: :+1:

File: src/TileChartMap.less (hunk 1 above)
Comment (flacoman91) [reply src/TileChartMap.less]: this is only for the legend bin values, and it was moved in the legend code here https://github.com/cfpb/ccdb5-ui/pull/191/files#diff-a7f3f10a59c833f0a7e2a676fcff76f4R288

For some reason IE11 wasn't respecting this css rule

**Pull request merged**